recipes/blake2/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class libb2Conan(ConanFile):
+    name = "libb2"
```

Comment:
@ericriff fix the package name and class name

Reply:
Please read the discussion above and give us your opinion on the naming situation. It is not clear what the name should be